scripts/get-operator-status.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -x
+
+if [ -z "${OPERATOR_NAMESPACE}" ]; then
+    echo "Please set OPERATOR_NAMESPACE"; exit 1
+fi
+
+if [ -z "$OPERATOR_NAME" ]; then
+    echo "Please set OPERATOR_NAME"; exit 1
+fi
+
+CSVNAME=$(oc get csv -n ${OPERATOR_NAMESPACE} -o jsonpath='{range .items[*]}{@.metadata.name}{"\n"}{end}' | grep $OPERATOR_NAME)
```

Comment:
Just checking for a single operator would not be enough I guess. dataplane-operator also installs ansibleee-operator, so just waiting for dataplane would not be sufficient.  So we need to keep all dependencies and check for them.

[1] https://github.com/openstack-k8s-operators/install_yamls/blob/main/Makefile#L1364

Reply:
I've updated the change to call the `make wait` target after operator installation. With this we ensure all operators installed via dependencies.